syntax = "proto3";

package ghidrasymbolserver;

service GhidraSymbolServer{
    rpc GetSymbolAtAddress(UnsignedLong) returns (String);
    rpc GetAddressOfSymbol(String) returns (UnsignedLong);
}

message UnsignedLong{
    uint64 value = 1;
}

message String{
    string value = 1;
}